From 04650764581bb2297a5dfdd9a15c53c63e5eeafd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Fri, 10 Jul 2015 13:02:00 +0200
Subject: [PATCH] Do not create Net/libnet.cfg
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

I will remove the Net/libnet.cfg because:

(1) it's content equals to default configuration hard-coded in the
code
(2) it's kind of configuration file we do not mark it as a configuration
file, so it's overwritten on each update
(3) it's loaded from directory based on Net::Config module location.
I.e. core module will search it in core path, vendor module in vendor
path and site module in site path.

perl.spec does not provide it either.

<https://bugzilla.redhat.com/show_bug.cgi?id=1238689>

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index 872eac8..750acc6 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -235,7 +235,7 @@ MAIN: {
 sub MY::post_initialize {
     my $self = shift;
 
-    return '' if $self->{PERL_CORE};
+    return '';
 
     if (not -f $CfgFile) {
         my @args = qw(Configure);
-- 
2.4.3

